src/lib/internal/helpers/typeahead.ts

```diff
@@ -51,15 +52,20 @@ export function createTypeaheadSearch(args: TypeaheadArgs = {}) {
 		typed.update(() => []);
 	});
 
-	const handleTypeaheadSearch = (key: string, items: HTMLElement[]) => {
-		if (ignoredKeys.has(key)) return;
+	const handleTypeaheadSearch = (e: KeyboardEvent, items: HTMLElement[]) => {
+		if (ignoredKeys.has(e.key) || isModifierKey(e)) return;
```

Comment:
I thought the same initially, but not quite. 

If you press `Ctrl`, the event is `{ key: "Control", ctrlKey: true }`: both `ignoredKeys` and `isModifierKey` will catch it.

If you press `Ctrl+a`, the event is `{ key: "a", ctrlKey: true }` and if you remove `isModifierKey`, this won't be caught since `ignoredKeys` checks `e.key` which is `a`. Even though `Ctrl` is pressed as well which we want to catch.

Reply:
Fair, nice catch!

Reply:
should I add a comment to make sure someone doesn't break in the future?